examples/splunk-sink/example.auto.tfvars

```diff
@@ -0,0 +1 @@
+../example.auto.tfvars
```

Comment:
I don't think that we should use the shared variables files for the splunk example since there's less overlap with the other examples. We only really need a `project_id` and a `credentials_path` so I'm in favor of switching this to a separate, non-symlinked file.

Reply:
haven't tackled this yet, but this was making it easier to run the 'apply_all.sh' and 'destroy_all.sh' to run all the examples.